src/external-server/docker-amarna.ts

```diff
@@ -33,8 +35,30 @@ export class AmarnaDocker extends DockerServer {
 
     public async run(args: { script?: boolean }) {
         this.useShell = !!args.script;
+        const formattedImage = `${this.image.repository}:${this.image.tag}`;
```

Comment:
The `pullImage` method was made protected but isn't used, although this looks like a use case for it.

Reply:
Aye, but it doesn't do it in sync, so we can't use it :(

Reply:
I don't understand, why `await this.pullImage(...)` be used?